packages/babel-traverse/src/visitors.js

```diff
@@ -106,7 +105,7 @@ export function explode(visitor) {
       if (existing) {
         mergePair(existing, fns);
       } else {
-        visitor[alias] = clone(fns);
+        visitor[alias] = { ...fns };
```

Comment:
@jayaddison Sorry I missed this while it was open but I think this could introduce an edge case behavior change if `fns` has any non-enumerable values in the prototype chain.  A better replacement might be:
```js
visitor[alias] = Object.create(Object.getPrototypeOf(fns));
Object.assign(visitor[alias], fns);
```

Here the cloned object is created with the prototype of `fns`, then `Object.assign` copies only the own iterable properties from fns.

Reply:
No problem at all, thanks for catching this.  Fixup opened in #11820.